### ISSUE TITLE
Time in UTC and not in Israel Time ??????

### ISSUE THREAD

**Issue by OmKoren** [MohGovIL/hamagen-react-native#157]
https://twitter.com/O_Koren/status/1245476089239089152

**Comment by ytzlax** [comment]
Maybe the next change in src/components/Main/ExposuresDetected.tsx file will fix the problem
```
        {`${weUnderstand}${Place} ${inDate} ${moment(fromTime).local().format('DD.MM.YY')} ${fromHour} ${moment(fromTime).local().format('HH:mm')}?`}

```

**Comment by MiriShalev** [comment]
Fixed in [V1.1.2 ](https://github.com/MohGovIL/hamagen-react-native/releases/tag/v1.1.2)
